### PR TITLE
Feature#854  Add COVID-19 information page 

### PR DESCRIPTION
**Pull request by kenson2998**
## Types of changes
- [ ] **Bugfix**
- [x] **New feature**
- [ ] **Refactoring**
- [ ] **Breaking change** (any change that would cause existing functionality to not work as expected)
- [ ] **Documentation Update**
- [ ] **Other (please describe)**

## Description

Add COVID-19 static pages.

## Steps to Test This Pull Request
1. Go to en-us or zh-hant  homepage.
2. Click on menu bar 『COVID-19 Guidelines』.
3. Click on footer list  『Privacy Policy』.

## Related Issue
[#854](https://github.com/pycontw/pycon.tw/issues/854)

## More Information
**Screenshots**
image
image
image

### CODE REVIEW

**Comment by mattwang44** [comment]
Do you miss the .po file with your commit? I do not see the trans works on your branch.
image

File: src/templates/pycontw-2020/contents/zh/guidelines.html [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+<hr>
+<p>
+	因應 2019 年新型冠狀病毒 (COVID-19) 疫情，「PyCon TW 主辦團隊」(以下簡稱「我們」) 參照我國中央流行疫情指揮中心的防疫措施，發布臺灣 Python 年會 (以下簡稱「PyCon TW」)
+	實體活動防疫守則。請您與我們一同遵守，保護自己與他人的健康。此防疫守則將視疫情發展與我國中央疫情疫情指揮中心指示有所變動，所有更動將不另通知，請在會前持續回來關心。最終版預計在 2020 年 08 月 20 日發布。
+</p>
+<hr>
+
+<h4>防疫要點</h4>
+
+<ol>
```
Comment (mattwang44) [inline src/templates/pycontw-2020/contents/zh/guidelines.html:20]: nit: It's better to have line spacing between the items in the ordered list. You can customize the style for it (ref: [page.scss](https://github.com/pycontw/pycon.tw/blob/master/src/static/pycontw-2020/styles/page.scss))

File: src/templates/pycontw-2020/contents/zh/about/privacy_policy.html [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+<p>我們為求本活動順利進行及提供您相關之服務，蒐集並於使用目的範圍內利用您所提供之個人資料，<b>其法定特定目的為一三〇會議管理</b>。使用目的包含但不限於：</p>
+<ol>
+	　
+	<li>確認報名者與前來報到者之身分同一性</li>
+	　
+	<li>提供您的名牌</li>
+	　
+	<li>其他有助於活動順利進行之相關目的</li>
+</ol>
```
Comment (mattwang44) [inline src/templates/pycontw-2020/contents/zh/about/privacy_policy.html:26]: excessive line break
image

```suggestion
<ol>
	<li>確認報名者與前來報到者之身分同一性</li>
	<li>提供您的名牌</li>
	<li>其他有助於活動順利進行之相關目的</li>
</ol>
```

File: src/templates/pycontw-2020/contents/zh/about/privacy_policy.html [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+<p>當您報名參與 PyCon TW 時，我們基於服務將會請您填寫服務所需之相關資料，而取得您的基本資料；我們僅將您的該些資料使用在您所需的個別服務上。</p>
+
+<h4>個人資料使用方式</h4>
+<p>我們將持續保存使用您提供的個人資料直至您提出停止使用或 PyCon TW 停止營運之日為止。我們<b>絕對不會</b>任意出售、交換、出租或以其他變相之方式，將您的個人資料揭露與其他團體或個人。惟有下列情形，我們會向第三者提供您的個人資料。</p>
+<ol>
+	　
+	<li>經過您的事前同意或授權允許時。</li>
+	　
+	<li>司法單位或其他主管機關經合法正式的程序要求時。</li>
+</ol>
```
Comment (mattwang44) [inline src/templates/pycontw-2020/contents/zh/about/privacy_policy.html:39]: Excessive line break
image

```suggestion
<ol>
	<li>經過您的事前同意或授權允許時。</li>
	<li>司法單位或其他主管機關經合法正式的程序要求時。</li>
</ol>
```

**Comment by kenson2998** [comment]
I fixed all comment tasks

File: src/static/pycontw-2020/styles/page.scss [hunk 4]
```diff
@@ -381,7 +381,16 @@ main {
...
+// customize privacy_policy and guidelines page ol
+ol.custom_ol {
+margin-top: 20px;
+}
 
+// customize privacy_policy and guidelines page h5 edited_info
+.edited_info{
+text-align: right;
+color: darkgray;
+}
```
Comment (mattwang44) [inline src/static/pycontw-2020/styles/page.scss:393]: please take care of ident & spacing. 
(p.s. consider adopting auto-formating tool, such as [Prettier](https://prettier.io/))
```suggestion
// customize privacy_policy and guidelines page ol
ol.custom_ol {
    margin-top: 20px;
}

// customize privacy_policy and guidelines page h5 edited_info
.edited_info {
    text-align: right;
    color: darkgray;
}
```

File: src/static/pycontw-2020/styles/page.scss [hunk 5]
```diff
@@ -381,7 +381,16 @@ main {
     }
 }
 
+// customize privacy_policy and guidelines page ol
+ol.custom_ol {
+margin-top: 20px;
```
Comment (mattwang44) [inline src/static/pycontw-2020/styles/page.scss:386]: I didn't make myself clear. I meant we can have spacing *between each item* in the ordered list, which slightly improves the readability.

File: src/static/pycontw-2020/styles/page.scss (hunk 4 above)
Comment (mattwang44) [inline src/static/pycontw-2020/styles/page.scss:393]: nit: I think we can have better naming for `custom_ol` & `edited_info`.

File: src/templates/pycontw-2020/contents/zh/guidelines.html [hunk 6]
```diff
@@ -76,11 +74,12 @@ <h4>PyCon TW 防疫準備</h4>
 </ol>
 
 <h4>個人資料保護聲明</h4>
+
 <ul>
 	<li>
 		PyCon TW 配合防疫期間所蒐集的資料，將在活動日起 28 天後刪除。參與者進出各館與各廳紀錄將作為疫情管控使用，若會後有任何疑似感染情況，PyCon TW
 		將主動通知可能被影響的群眾，且得提供我國衛生主管機關依傳染病防治法等規定進行疫情調查及聯繫使用。詳細聲明請參照：<a
-		href="https://docs.google.com/document/d/1wcTCXs75VxRK2xWdd0lKD57HCMp7ts_MT7lNSJ9_euU/edit#">PyCon Taiwan
+		href="../about/privacy_policy/">PyCon Taiwan
```
Comment (mattwang44) [inline src/templates/pycontw-2020/contents/zh/guidelines.html:82]: this indeed functional but we prefer using [`url` tag](https://docs.djangoproject.com/en/3.0/ref/templates/builtins/#url) (ref: [warmup page](https://github.com/pycontw/pycon.tw/blob/master/src/templates/pycontw-2020/contents/en/events/warmup-session.html#L9-L11)).

File: src/static/pycontw-2020/styles/page.scss (hunk 5 above)
Comment (kenson2998) [reply src/static/pycontw-2020/styles/page.scss:386]: okay , I got it .

File: src/static/pycontw-2020/styles/page.scss (hunk 4 above)
Comment (kenson2998) [reply src/static/pycontw-2020/styles/page.scss:393]: I have no idea about naming , so I create a temp name for it . 🤔

File: src/templates/pycontw-2020/contents/zh/guidelines.html (hunk 6 above)
Comment (kenson2998) [reply src/templates/pycontw-2020/contents/zh/guidelines.html:82]: got it .

File: src/static/pycontw-2020/styles/page.scss (hunk 4 above)
Comment (mattwang44) [reply src/static/pycontw-2020/styles/page.scss:393]: ok, we can leave it like this for now. at least you put the clear comment :).

File: src/templates/pycontw-2020/_includes/menu.html [hunk 7]
```diff
@@ -26,6 +26,8 @@
 {% url 'page' path='sponsor/sponsor' as sponsor_url %}
 {% url 'page' path='events/warmup-session' as warmup_session %}
 {% url 'page' path='events/tutorials' as events_tutorial_list_url %}
+{% url 'page' path='guidelines' as guidelines_url %}
```
Comment (yychen) [inline src/templates/pycontw-2020/_includes/menu.html:29]: 這邊有個建議，把東西移到 'covid-19/guidelines' 底下。

File: src/templates/pycontw-2020/_includes/menu.html [hunk 8]
```diff
@@ -111,11 +113,19 @@
...
 		</div>
 	</li>
+
 	<li>
 		<a href="{{ venue_url }}" {% if category == 'venue' %}class="active" {% endif %}>
 			{% trans 'Venue' context 'conf location' %}
 		</a>
 	</li>
+	<li>
+		<a href="{{ guidelines_url }}" {% if category == 'COVID-19 Guidelines' %}class="active" {% endif %}>
```
Comment (yychen) [inline src/templates/pycontw-2020/_includes/menu.html:123]: 這邊 category == 'covid-19' 點選它的時候就會有底線。詳細可以 trace 一下 code。

Comment (mattwang44) [reply src/templates/pycontw-2020/_includes/menu.html:123]: ref: #696

Comment (yychen) [reply src/templates/pycontw-2020/_includes/menu.html:123]: 天啊年代久遠我都忘記是你做的了 orz

File: src/templates/pycontw-2020/contents/en/about/privacy_policy.html [hunk 9]
```diff
@@ -73,7 +73,9 @@ <h4>Your Rights of Personal Information</h4>
 	<li>Any requests to make duplications of the personal information</li>
 	<li>Any requests to add supplements or correct the personal information</li>
 	<li>Any requests to discontinue the collection, processing or using of personal information</li>
-	<li>Any requests to delete the personal information unless you do not agree the process of registration and legal regulation</li>
+	<li>Any requests to delete the personal information unless you do not agree the process of registration and legal
```
Comment (mattwang44) [inline src/templates/pycontw-2020/contents/en/about/privacy_policy.html:76]: is this line break necessary?

File: src/templates/pycontw-2020/contents/en/covid-19/guidelines.html [hunk 10]
```diff
@@ -109,15 +109,15 @@ <h4>Privacy Policy of PyCon Taiwan</h4>
...
 </ul>
 
 
 <h4>License (CC BY-SA 3.0 TW)</h4>
 <p>
-	This document is licensed under a <a href="https://creativecommons.org/licenses/by-sa/3.0/tw/deed.zh_TW">Attribution-ShareAlike
+	This document is licensed under a <a target="_blank" rel="noopener" href="https://creativecommons.org/licenses/by-sa/3.0/tw/deed.zh_TW">Attribution-ShareAlike
 	3.0 Taiwan</a> license.
 </p>
 <p><img src="/static/images/CC_icon.png"></p>
```
Comment (mattwang44) [inline src/templates/pycontw-2020/contents/en/covid-19/guidelines.html:123]: should also use `static` here

File: src/templates/pycontw-2020/contents/zh/covid-19/guidelines.html [hunk 11]
```diff
@@ -87,13 +87,13 @@ <h4>個人資料保護聲明</h4>
...
+	<li><a target="_blank" rel="noopener" href="https://drive.google.com/file/d/1Gt3KgRFNgs-PBB37zHU_CiweqTngXjL_/view?usp=sharing">教育部函「防疫新生活運動」室內外集會須配合中央疫情指揮中心發布之相關指引</a>
 	</li>
-	<li><a href="http://serv.oga.ncku.edu.tw/var/file/57/1057/img/1292/104982743.pdf">國立成功大學因應嚴重特殊傳染性肺炎公共空間管理原則</a></li>
+	<li><a target="_blank" rel="noopener" href="http://serv.oga.ncku.edu.tw/var/file/57/1057/img/1292/104982743.pdf">國立成功大學因應嚴重特殊傳染性肺炎公共空間管理原則</a></li>
 </ul>
 
 <h4>授權 License (CC BY-SA 3.0 TW)</h4>
-<p>此文件採用 <a href="https://creativecommons.org/licenses/by-sa/3.0/tw/deed.zh_TW">姓名標示-相同方式分享 3.0 台灣</a> 授權條款。</p>
+<p>此文件採用 <a target="_blank" rel="noopener" href="https://creativecommons.org/licenses/by-sa/3.0/tw/deed.zh_TW">姓名標示-相同方式分享 3.0 台灣</a> 授權條款。</p>
 <p><img src="/static/images/CC_icon.png"></p>
```
Comment (mattwang44) [inline src/templates/pycontw-2020/contents/zh/covid-19/guidelines.html:97]: should also use `static` here

File: src/templates/pycontw-2020/contents/en/about/privacy_policy.html [hunk 12]
```diff
@@ -0,0 +1,127 @@
...
+</p>
+<p>
+	PyCon TW 2020 collects personal data of attendees for the purpose of contact tracing during the outbreak of COVID-19.
+	All collected information will be deleted after 28 days. attendees’ access in each venue will be recorded for epidemic
+	control measures. We will inform everyone that if any suspected COVID-19 and provide such information for CECC for the
+	purpose of preventing spread of COVID-19 and tracing contacts.
+</p>
+
+<h4>License (CC BY-SA 3.0 TW)</h4>
+<p>This document is licensed under a <a target="_blank" rel="noopener" href="https://creativecommons.org/licenses/by-sa/3.0/tw/deed.zh_TW">Attribution-ShareAlike
```
Comment (yychen) [inline src/templates/pycontw-2020/contents/en/about/privacy_policy.html:116]: I'll approve this PR first, but after this, I'd suggest change this link into the English version https://creativecommons.org/licenses/by-sa/3.0/tw/deed.en

Sorry I didn't notice this last time.

File: src/templates/pycontw-2020/contents/en/covid-19/guidelines.html [hunk 13]
```diff
@@ -0,0 +1,132 @@
...
+<ul class="custom_ul">
+	<li><a target="_blank" rel="noopener" href="https://drive.google.com/file/d/1Gt3KgRFNgs-PBB37zHU_CiweqTngXjL_/view?usp=sharing">教育部函「防疫新生活運動」室內外集會須配合中央疫情指揮中心發布之相關指引</a>
+	</li>
+	<li><a target="_blank" rel="noopener" href="http://serv.oga.ncku.edu.tw/var/file/57/1057/img/1292/104982743.pdf">國立成功大學因應嚴重特殊傳染性肺炎公共空間管理原則</a></li>
+</ul>
+
+
+<h4>License (CC BY-SA 3.0 TW)</h4>
+<p>
+	This document is licensed under a <a target="_blank" rel="noopener" href="https://creativecommons.org/licenses/by-sa/3.0/tw/deed.zh_TW">Attribution-ShareAlike
```
Comment (yychen) [inline src/templates/pycontw-2020/contents/en/covid-19/guidelines.html:120]: Same here.

File: src/templates/pycontw-2020/contents/en/about/privacy_policy.html (hunk 12 above)
Comment (kenson2998) [reply src/templates/pycontw-2020/contents/en/about/privacy_policy.html:116]: ok~

**Pull request merged**